index.html

```diff
@@ -1,12 +1,921 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
-    <title>Document</title>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="icon" type="image/png" href="assets/icons/Symbol-New-York-Times.png" />
+  <link rel="stylesheet" href="styles/app.css" />
+  <link href="https://www.fontify.me/wf/1cc89de3d33c62b9e9d8cdb0afa715aa" rel="stylesheet" type="text/css" />
+  <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+  <title>Space Ripples Reveal Big Bangs Smoking Gun</title>
 </head>
+
 <body>
-    <h1>hello world</h1>
+  <header>
+    <div class="nav-container">
+      <ul class="left-nav flex flex-align">
+        <li><i class="fas fa-bars fa-xs"></i></li>
+        <li><i class="fas fa-search fa-xs"></i></li>
+      </ul>
+      <img src="assets/img/ny-logo.png" alt="main-logo" />
```

Comment:
please wrap the new york times logo with an anchor tag. cause it should be a link